testing/environments/5.x.yml

```diff
@@ -0,0 +1,33 @@
+# This is the latest stable 5x release environment.
```

Comment:
This directory is copied directly from beats. We don't need that file, but I'd rather just copy everything rather than cherry-pick and make the vendoring more complicated. WDYT?

Reply:
sounds good